.github/scripts/end2end/generate-kustomization.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Constants for valid topologies for CI tests
+# We support multiple shards per host, or one shard per host, up to 9 nodes
```

Comment:
please document "what" is such topology: i.e. what are the first and last numbers...